src/components/EditBookModal.jsx

```diff
@@ -0,0 +1,265 @@
+import React from "react";
+import Modal from "react-modal";
+import CloseButton from "../helpers/CloseButton";
+import { motion } from "framer-motion";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+
+const el = document.getElementById("root");
+Modal.setAppElement(el);
+
+const validationSchema = Yup.object().shape({
+  BookName: Yup.string()
+    .min(2, "Too Short!")
+    .max(50, "Too Long!")
+    .required("Required"),
+  Author: Yup.string()
+    .min(2, "Too Short!")
+    .max(50, "Too Long!")
+    .required("Required"),
+  Genre: Yup.string().required("Required"),
+  Price: Yup.number().required("Required"),
+});
+
+const initialValues = {
+  BookName: "",
+  Author: "",
+  Genre: "",
+  Price: "",
+  Description: "",
+  Image: "",
+};
+
+function ContactModal({ isEditBookOpen, setIsEditBookOpen }) {
```

Comment:
Same thing here, when you edit the data you expect previous info to be there and you edit them. So how can we send the data to this modal?